components/profile_popover/profile_popover.tsx

```diff
@@ -400,45 +380,42 @@ class ProfilePopover extends React.PureComponent {
                 </div>,
             );
         }
-
         if (this.props.user.is_bot && !haveOverrideProp) {
             dataContent.push(
                 <div
                     key='bot-description'
                     className='overflow--ellipsis text-nowrap'
                 >
-                    {this.props.user.bot_description}
+                    {this.props.user.bot_description as ReactNode}
                 </div>,
             );
         }
-
         if (this.props.user.position && !haveOverrideProp) {
-            const position = this.props.user.position.substring(0, Constants.MAX_POSITION_LENGTH);
+            const position = (this.props.user?.position as string).substring(
+                0,
+                Constants.MAX_POSITION_LENGTH,
+            );
             dataContent.push(
                 <OverlayTrigger
                     delayShow={Constants.OVERLAY_TIME_DELAY}
                     placement='top'
                     overlay={<Tooltip id='positionTooltip'>{position}</Tooltip>}
                     key='user-popover-position'
                 >
-                    <div
-                        className='overflow--ellipsis text-nowrap pt-1 pb-1'
-                    >
+                    <div className='overflow--ellipsis text-nowrap pt-1 pb-1'>
                         {position}
                     </div>
                 </OverlayTrigger>,
             );
         }
-
-        const email = this.props.user.email;
+        const email = this.props.user.email as string;
```

Comment:
Same here wrt. coalescing to blank string.

Reply:
Done